src/Predict/JobForm/SegmentationForm.js

```diff
@@ -7,6 +7,33 @@ import Typography from '@mui/material/Typography';
 import ResolutionDropdown from './ResolutionDropdown';
 import ChannelForm from './ChannelForm';
 import jobData from '../jobData';
+import FormControl from '@mui/material/FormControl';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+
+function DimensionOrderDropdown(props) {
+  const { value, onChange } = props;
+  const orders = ['bxy', 'cxy', 'bxyc', 'cxyb'];
```

Comment:
I'm wondering if these options would look better in all caps in the dropdown menu, but that is clearly an subjective decision so I'll defer to your preference.

Reply:
I think it should be all caps too. I've been using OME-TIFF's dimension order as a comparison point and they use all caps

https://docs.openmicroscopy.org/ome-model/6.0.0/ome-tiff/specification.html#dimensionorder